twilio/http/validation_client.py

```diff
@@ -1,4 +1,7 @@
-from urlparse import urlparse
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse
```

Comment:
We might want to do this in the `compat.py` file and then import `urlparse` from that so this file doesnt have to care which version of python its being run under.